kore/src/Kore/Unparser.hs

```diff
@@ -57,50 +62,81 @@ import qualified Numeric
  -}
 class Unparse p where
     -- | Render a type from abstract to concrete Kore syntax.
+    -- original/old unparser
     unparse :: p -> Doc ann
+    -- applicative/new unparser
+    unparse2 :: p -> Doc ann
```

Comment:
This function name is uninformative. Could you rename it to somehing like unparseToApplicative? Renaming the original to unparseToKore might also be nice (or maybe toApplicative and toKore).

The same for the other "*2" names.